msi/rust.wxs

```diff
@@ -192,9 +201,9 @@
                         <Shortcut Id="RustShell"
                                   Name="$(var.ProductName) Shell"
                                   Description="Opens Command Prompt with Rust tools directory added to the PATH"
-                                  Target="[SystemFolder]CMD.exe"
+                                  Target="[SHORTCUTTARGET]"
                                   Arguments="/K path [INSTALLDIR]bin;%PATH%"
-                                  WorkingDirectory="PersonalFolder">
+                                  WorkingDirectory="SHORTCUTWKDIR">
```

Comment:
Huh, didn't know that.  That's annoying. I wonder if there's some way to use an extra level of indirection to avoid this expansion...
Otherwise, lgtm.